README.md

```diff
@@ -111,6 +111,16 @@ Currently Java 8 and Java 11 are bundled (JDK_VERSION= {8, 11}).
 
 Full list of options for JDK11 can be found [here](./Makefile).
 
+#### Running PCT with custom Java versions in Docker
+
+When using the docker image, it is possible to use `JDK_VERSION` variable to select 
+the version to use. The version needs to be bundled in the docker image. By specifying 
+`11`, the modules and jaxb libraries are also added to the command line. 
```

Comment:
The JAXB library is no more needed after the detached plugin. We can rephrase the last part of the paragraph.